src/Catalogue/BookInstanceAddedToCatalogue.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Akondas\Library\Catalogue;
+
+use Akondas\Library\Common\Event\DomainEvent;
+use Akondas\Library\Common\UUID;
+use DateTimeImmutable;
+
+class BookInstanceAddedToCatalogue implements DomainEvent
+{
+    private UUID $eventId;
+
+    private string $isbn;
+
+    private BookType $bookType;
+
+    private UUID $bookId;
+
+    private DateTimeImmutable $when;
+
+    /**
+     * BookInstanceAddedToCatalogue constructor.
+     */
```

Comment:
I think you need to edit your phpstorm class template :wink:

Reply:
Yeah I need 😄